custom_components/wellbeing/api.py

```diff
@@ -240,14 +246,24 @@ def setup(self, data):
 
     @property
     def speed_range(self) -> tuple:
+        ## Electrolux Devices: 
+        if self.model == "WELLA5":
+            return 1, 5
         if self.model == "WELLA7":
             return 1, 5
         if self.model == "AX7":
```

Comment:
Duplicate. Please remove it.

Reply:
where is the duplicate? 

the current code of your branch:
<img width="196" alt="image" src="https://github.com/JohNan/homeassistant-wellbeing/assets/47820557/c7282c6d-0335-4555-a31d-05fead22087a">

Reply:
See your diff https://github.com/JohNan/homeassistant-wellbeing/blob/1868cbb0f6eb45df24377ee9809d0e2420346118/custom_components/wellbeing/api.py#L254-L263